streams/src/test/java/org/apache/kafka/streams/tests/StreamsStandByReplicaTest.java

```diff
@@ -110,6 +111,8 @@ public String apply(final Long value) {
             }
         };
 
+        inputStream.print(Printed.<String, String>toSysOut());
```

Comment:
Is this for debugging only or is it necessary for the test itself?

Reply:
Ack. It's left over from debugging; I'll remove it.